tests/core/creators/manual/resizewithcssunits.md

```diff
@@ -0,0 +1,39 @@
+@bender-tags: bug, 4.14.1, 1883
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, table
+
+## Scenario 1
+
+1. In Width input type `20cm`.
```

Comment:
It would be nice to add additional units to test - the most important check out if this feature still works with number units. You can get a list of absolute units from https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units

A simple list with these units and information to use them for calculation (by repeating test steps) should be enough.